lib/dor/text_extraction/ocr.rb

```diff
@@ -48,6 +48,13 @@ def cleanup
           FileUtils.rm_f(abbyy_results.result_path)
         end
 
+        # e.g. /abbyy/EXCEPTIONS/druid:ab123cd4567.xml.result.xml
+        abbyy_exceptions = Dir.glob("#{Settings.sdr.abbyy.local_exception_path}/*#{bare_druid}*")
```

Comment:
idk if we expect this dir to only contain XML tickets for exceptions.  but either way, maybe this glob should include the `.xml` extension (so e.g. `*#{bare_druid}*.xml`), just as an extra safeguard?

Reply:
yes good call...

there will possibly be image files that are bad also here, those will remain (though the general cleanup rake task will remove anything older than 1 month if run)